src/main/java/com/gmail/chernii/oleksii/list/MyArrayList.java

```diff
@@ -0,0 +1,64 @@
+package com.gmail.chernii.oleksii.list;
+
+import java.util.Arrays;
+
+/**
+ * Created by Space on 05.03.2019.
+ */
+public class MyArrayList<E> implements List<E> {
+    private E[] elements;
+    private static final int DEFAULT_LENGTH = 10;
```

Comment:
firstly declare static field, then non-static
[Ordering of class contents](https://mate-academy.github.io/style-guides/java/java.html#s3.4.2-ordering-class-contents)